pyiso/isone.py

```diff
@@ -27,6 +28,17 @@ class ISONEClient(BaseClient):
         'Landfill Gas': 'biogas',
     }
 
+    locations = {
```

Comment:
Simpler location data structure. Does this have all the items we need (just the 'Z' ones)?